shelf_router/test/router_test.dart

```diff
@@ -148,6 +148,30 @@ void main() {
     expect(await get('/api/hello?ok'), 'middleware');
   });
 
+  test('mount(Router) does not require a trailing slash', () async {
+    var api = Router();
+    api.get('/', (Request request) {
+      return Response.ok('Hello World!');
+    });
+
+    var app = Router();
+    app.get('/hello', (Request request) {
+      return Response.ok('hello-world');
+    });
+
+    app.mount('/api', api);
+
+    app.all('/<_|[^]*>', (Request request) {
+      return Response.ok('catch-all-handler');
+    });
+
+    server.mount(app);
+
+    expect(await get('/hello'), 'hello-world');
+    expect(await get('/api'), 'Hello World!');
```

Comment:
Both of these should work, right?
```suggestion
    expect(await get('/api'), 'Hello World!');
    expect(await get('/api/'), 'Hello World!');
```

Reply:
yup just added the additional expectation 👍